src/api/apiconfig_api/v1/_base_policy.xml.tpl

```diff
@@ -28,6 +28,7 @@
             </allowed-headers>
         </cors>
         <base />
+        <rate-limit calls="90" renewal-period="90"/>
```

Comment:
@jacopocarlini to avoid general throttling, could be useful evaluate this config https://docs.microsoft.com/en-us/azure/api-management/api-management-sample-flexible-throttling#user-identity-throttling ?

Reply:
done!